docs/source/asynchronous.rst

```diff
@@ -80,30 +81,24 @@ Example
 -------
 
 This self-contained example starts an asynchronous client, submits a trivial
-job, waits on the result, and then shuts down the client. You can see
-implementations for Asyncio and Tornado.
+job, waits on the result, and then shuts down the client.
 
-Python 3 with Tornado or Asyncio
-++++++++++++++++++++++++++++++++
+Python with Asyncio
++++++++++++++++++++
 
 .. code-block:: python
 
+   import asyncio
+
    from dask.distributed import Client
 
    async def f():
-       client = await Client(asynchronous=True)
-       future = client.submit(lambda x: x + 1, 10)
-       result = await future
-       await client.close()
+       async with Client(asynchronous=True) as client:
+          future = client.submit(lambda x: x + 1, 10)
+          result = await future
        return result
 
-   # Either use Tornado
-   from tornado.ioloop import IOLoop
-   IOLoop().run_sync(f)
```

Comment:
the "Either use Tornado" "Or use asyncio" is redundant now that `asyncio.run` is the recommended approach for starting tornado:

https://github.com/tornadoweb/tornado/blob/3de78b7a15ba7134917a18b0755ea24d7f8fde94/README.rst#hello-world